libs/wire-api/src/Wire/API/Routes/Bearer.hs

```diff
@@ -35,6 +35,10 @@ instance FromHttpApiData a => FromHttpApiData (Bearer a) where
     _ -> Left "Invalid authorization scheme"
   parseUrlPiece = parseHeader . T.encodeUtf8
 
+instance ToHttpApiData a => ToHttpApiData (Bearer a) where
```

Comment:
what is this needed for?  there may be security implications if this is passed in the path or as a query param.  is there a way to avoid that?  (or is it "best practice" to do it this way?

Reply:
also, if you keep it please add a roundtrip-test.

Reply:
It's only needed in test, but yes, we can make the production code safer by moving this to the test code.